README.md

```diff
@@ -18,7 +18,7 @@
 
 The best and easiest way to start coding standard with. Combined both [PHP-CS-Fixer](https://github.com/friendsofphp/php-cs-fixer) and [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer).
 
-**Used by [Nette](https://github.com/nette/coding-standard) and [Sylius](https://github.com/SyliusLabs/CodingStandard).**
+**Used by [Shopsys](https://github.com/shopsys/coding-standards), [Nette](https://github.com/nette/coding-standard) and [Sylius](https://github.com/SyliusLabs/CodingStandard).**
```

Comment:
It'd be easier to review if such changes were separated in another PR since they aren't connected with the problem this PR is solving. Just a hint 🤓

Reply:
Hint taken :+1: 

It was just detail docs addition, where extra PR would not make sense (no code changes nor changelog note etc.).

Sorry if that confused you during review.

Reply:
no problem!